main.go

```diff
@@ -54,15 +54,15 @@ func main() {
 	)
 
 	var stager stage.Stager
-
-	stager = &s3.Stager{
-		Bucket: cfg.StageBucket,
-	}
-
-	if cfg.MinioDev {
+	if os.Getenv("CLOWDER_ENABLED") == "true" {
 		stager = minio.GetClient(&minio.Stager{
```

Comment:
If `CLOWDER_ENABLED==true` in prod, the object store may not be minio so I think the logic needs to change here.

Actually I don't think `CLOWDER_ENABLED` matters here. If object store is mino, use minio client... else, use s3 client -- does the cdappconfig.json tell us what the object store provider type is?

Reply:
This is for some backwards compatibility. The plan going forward is to always use minio as far as I understand. You do bring up a good point about prod though. I'm not entirely sure.